### PR TITLE
Speed up chart rendering by caching styles, themes & visualizations

### PR DESCRIPTION
**Pull request by davidkokkelink**
TLDR: Make charts render **50-70% faster** by accessing the cache directly from the `frontend` service for visualization meta, themes & computed CSS.

----

On every chart render, the `preview` endpoint makes 4 API calls:
1. to `/chart/{id}/publish/data` to get the data for the chart
2. to `/visualizations/{vis.id}` to get information about the visualization
3. to `/visualizations/{vid.id}/style.css?theme={themeId}` to get the theme-vis css
4. to `/themes/{theme.id}` to get theme properties

Only the first request actually changes frequently. This PR changes it so that requests 2-4 are served from the redis cache if possible. In fact, 3 was already written to the redis cache in a prior optimization, so this PR only changes it so that the `frontend` service looks it up directly from the cache rather than having the API look in the cache. For 2 & 4, the additional cache segments `visualizations` and `themes` are introduced, which are simple key-value stores that do what you'd expect.

This also depends on this [PR in plugin-themes](https://github.com/datawrapper/plugin-themes/pull/16) which adds invalidation for the `themes` cache.

### CODE REVIEW

**Comment by gka** [comment]
Really nice!

**Comment by davidkokkelink** [comment]
Changed the implementation to use `config.general.cache`
